### PR TITLE
LoadErrorMessage component

### PR DESCRIPTION
**Pull request by syedsalehinipg**
Contributes to [TD-2030](https://sce.myjetbrains.com/youtrack/issue/TD-2030)

## Changes

- Implemented a flexible and customizable error message component named LoadErrorMessage.
- Incorporated a toggleable "View More Details" section to reveal additional error information when provided.
- Integrated an optional action button that executes a specified callback function when clicked.
- Added a "Contact Team" section with an optional support URL that opens in a new tab when clicked.

Figma:
https://www.figma.com/file/s2NuBwBayu8144EGjBCi5d/VIRTO.DATA?node-id=281%3A22633&mode=dev
_(under Error examples)_

## UI/UX

image

image

image

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
- [ ] Branch has been run in docker.
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [x] Appropriate tests have been added.
- [x] Lint and test workflows pass.

### CODE REVIEW

**Comment by m4manjesh** [comment]
@syedsalehinipg is there any Figma?

**Comment by syedsalehinipg** [comment]
> @syedsalehinipg is there any Figma?

Yes, please find the URL from in the description.

**Comment by Sowbhagya-ipg** [comment]
@syedsalehinipg  We need to work on the dark theme for these Components. I will quickly update the mockup for you to work on it.

**Review by m4manjesh — CHANGES_REQUESTED** [review]
Some code changes required and a question. See comments.

File: src/LoadErrorMessage/LoadErrorMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            variant="body2"
+            color="var(--light-primary-main, #003063)"
+            style={{ cursor: "pointer" }}
+            onClick={handleDetailsClick}
+          >
+            Hide More Details
+          </Typography>
+          {/* Render the error details */}
+          <Box
+            display="flex"
```
[the remark this answers is not in the document]
Comment (syedsalehinipg) [reply src/LoadErrorMessage/LoadErrorMessage.tsx]: I am not sure if the Snackbar is the right one for this but I have used the Alert instead

File: src/LoadErrorMessage/LoadErrorMessage.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { expect, test } from "@playwright/test";
+
+test("Renders the title", async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (syedsalehinipg) [reply src/LoadErrorMessage/LoadErrorMessage.spec.tsx:3]: That was my understanding, @VictorCLLucas can you please confirm?

[the remark this answers is not in the document]
Comment (m4manjesh) [reply src/LoadErrorMessage/LoadErrorMessage.spec.tsx:3]: We should always go with a better and easy to use one, so here i am okay with Playwright.

**Review by m4manjesh — APPROVED** [review]
LGTM

**Pull request merged**